azure-pipeline/build-artifacts-and-some-tests-job-steps.yaml

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+steps:
+    - template: ./install-dependencies-steps.yaml
+
+    - script: |
+          yarn format:check
+      displayName: check code formatting
+
+    - script: |
+          yarn copyright:check
+      displayName: check for copyright headers
+
+    - template: ./build-steps.yaml
+
+    - script: |
+          yarn lint:check
+      displayName: run linter
+
+    - template: ./test-steps.yaml
+      parameters: { totalSlices: '${{ parameters.totalSlices }}', slicesToRun: '${{ parameters.slicesToRun }}' }
```

Comment:
what is `slice`?  can we have some generic message?

Reply:
I am slicing the tests in to n numbers & then run the project tests that belong to the slice numbers specified.

Reply:
any suggestion on naming?

Reply:
never mind